onecore_maintenance_extension/models/maintenance.py

```diff
@@ -169,40 +168,36 @@ def update_form_options(self, search_by_number, search_type):
                             'email_address': tenant['emailAddress'],
                             'phone_number': phone_number,
                             'is_tenant': tenant['isTenant'],
-                          })
+                        })
         else:
             _logger.info("No data found in response.")
 
-    @api.depends('search_by_number', 'search_type')
+    @api.onchange('search_by_number')
```

Comment:
Do we want this to trigger when `search_type` changes too?

Reply:
Yes, of course 🙏